src/main/java/org/elasticsearch/index/query/MatchAllQueryBuilder.java

```diff
@@ -64,15 +56,19 @@ public MatchAllQueryBuilder boost(float boost) {
         return this;
     }
 
+    /**
+     * @return the boost for this query.
+     */
+    public float getBoost() {
```

Comment:
I also thought about making this package private, only used in test so far. Changing setter is also fine with me but will break client code somewhere I guess?